<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWordDialog</class>
 <widget class="QDialog" name="EditWordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="m_newWordStringLineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="m_newWordWeightSpinBox">
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="m_anulujButton">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="m_updateButton">
     <property name="text">
      <string>Aktualizuj</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
